<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>拧紧枪选择</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxS1">
            <property name="text">
             <string>1号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxS2">
            <property name="text">
             <string>2号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxS3">
            <property name="text">
             <string>3号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBoxS4">
            <property name="text">
             <string>4号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxS5">
            <property name="text">
             <string>5号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBoxS6">
            <property name="text">
             <string>6号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBoxS7">
            <property name="text">
             <string>7号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBoxS8">
            <property name="text">
             <string>8号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBoxS9">
            <property name="text">
             <string>9号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBoxS10">
            <property name="text">
             <string>10号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBoxS11">
            <property name="text">
             <string>11号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="checkBoxS12">
            <property name="text">
             <string>12号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="checkBoxS13">
            <property name="text">
             <string>13号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="checkBoxS14">
            <property name="text">
             <string>14号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="checkBoxS15">
            <property name="text">
             <string>15号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QCheckBox" name="checkBoxS16">
            <property name="text">
             <string>16号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="checkBoxS17">
            <property name="text">
             <string>17号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QCheckBox" name="checkBoxS18">
            <property name="text">
             <string>18号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QCheckBox" name="checkBoxS19">
            <property name="text">
             <string>19号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QCheckBox" name="checkBoxS20">
            <property name="text">
             <string>20号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QCheckBox" name="checkBoxS21">
            <property name="text">
             <string>21号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QCheckBox" name="checkBoxS22">
            <property name="text">
             <string>22号枪</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QPushButton" name="pushButtonSelectReverse">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>反选</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QPushButton" name="pushButtonSelectAll">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>全选</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>开始日期</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditStart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="displayFormat">
          <string>yyyy/M/d</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>结束日期</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditEnd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="displayFormat">
          <string>yyyy/M/d</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxAllData">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>读取全部数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dateEditStart</tabstop>
  <tabstop>checkBoxS22</tabstop>
  <tabstop>checkBoxS21</tabstop>
  <tabstop>checkBoxS20</tabstop>
  <tabstop>checkBoxS19</tabstop>
  <tabstop>checkBoxS18</tabstop>
  <tabstop>checkBoxS17</tabstop>
  <tabstop>checkBoxS16</tabstop>
  <tabstop>checkBoxS15</tabstop>
  <tabstop>checkBoxS14</tabstop>
  <tabstop>checkBoxS13</tabstop>
  <tabstop>checkBoxS12</tabstop>
  <tabstop>checkBoxS11</tabstop>
  <tabstop>checkBoxS10</tabstop>
  <tabstop>checkBoxS9</tabstop>
  <tabstop>checkBoxS8</tabstop>
  <tabstop>checkBoxS7</tabstop>
  <tabstop>checkBoxS6</tabstop>
  <tabstop>checkBoxS5</tabstop>
  <tabstop>checkBoxS4</tabstop>
  <tabstop>checkBoxS3</tabstop>
  <tabstop>checkBoxS2</tabstop>
  <tabstop>checkBoxS1</tabstop>
  <tabstop>dateEditEnd</tabstop>
  <tabstop>pushButtonSelectReverse</tabstop>
  <tabstop>checkBoxAllData</tabstop>
  <tabstop>pushButtonSelectAll</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAllData</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateEditEnd</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAllData</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateEditStart</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
